pkg/api/types.go

```diff
@@ -1190,6 +1190,8 @@ type ServiceSpec struct {
 	// Type determines how the service will be exposed.  Valid options: ClusterIP, NodePort, LoadBalancer
 	Type ServiceType `json:"type,omitempty"`
 
+	LoadBalancerIP string `json:"loadBalancerIP, omitEmpty`
```

Comment:
@thockin the answer to the AWS question in no; the ELB name is assigned by AWS.  If we use Route 53 (which maybe we should) we can create a DNS record pointing to that ELB.